share/eds/inverted_index.go

```diff
@@ -14,6 +15,9 @@ import (
 
 const invertedIndexPath = "/inverted_index/"
 
+// ErrNotFoundInIndex is returned instead of ErrNotFound if the multihash doesn't exist in the index
+var ErrNotFoundInIndex = fmt.Errorf("failed to lookup index")
```

Comment:
this error message is confusing - shouldn't it be "does not exist in index" then?